src/tests/webloader.test.js

```diff
@@ -188,6 +188,35 @@ describe('the web loader', () => {
 		assert.ok(scripts[0].src.indexOf(MOCK_GA_CLIENT_ID) !== 0, 'The uid should have been set.');
 	});
 
+	it('should use generated uid if GA integration times out', async () => {
+		setupGlobal(null, undefined, { evolvLazyUid: 'true' });
+
+		// overwrite ga.getAll to force integration polling method to time out
+		window.ga.getAll = () => [];
+
+		const prefix = '11';
+		const ticks = 60;
+		const MOCK_GENERATE_UID = `${prefix}_${ticks}`;
+
+		// used to mock the prefix of the generated uid
+		let mathRoundStub = sinon.stub(Math, 'round').returns(prefix);
```

Comment:
It might make sense to mock `generate()` instead of `Math.round()`. Your test is limited in scope, true, but who knows what else might use `Math.round()` under the hood.

Reply:
@mattstrom - yeah, I was having a hard time trying to use Sinon to mock the generate method compared to Jest. I've done it a million times in Jest, but when I try using Sinon I get an error that can't mock or spy on an ES module.

Reply:
I see. We can stick with `Math.round()` then.

Yeah, I would love for this repo to use Jest.

Reply:
That would be awesome!